### ISSUE TITLE
✘ [ERROR] Expected value for define "process.env.NODE_ENV" to be a string, got undefined instead

### ISSUE THREAD

**Issue by johanguse** [sdorra/content-collections#23]
Hey guys! I am trying to move away from Contentlayer, but I can't build the content anyone knows why?
I've used bun and pnpm, and both got the error below.

```
projectfolder/node_modules/.pnpm/esbuild@0.19.12/node_modules/esbuild/lib/main.js:300:10:
      300 │     throw new Error(`Expected value for ${what}${key !== void 0 ? " " + quote(key) : ""} to be a string, got ${typeof value} ...
          ╵           ^
```
command: "build:content": "content-collections build"

Try with node 19, 20 and 21 
node v21.6.1 (npm v10.2.4)

I also use @t3-oss/env-nextjs

content-collections.ts

```
import { defineCollection, defineConfig } from '@content-collections/core'
import { compileMDX } from '@content-collections/mdx'

const posts = defineCollection({
  name: 'posts',
  directory: 'content/blog',
  include: '*.mdx',
  schema: (z) => ({
    title: z.string(),
    description: z.string(),
    date: z.string(),
    image: z.string(),
    authors: z.array(z.string()),
    published: z.boolean().default(true),
    tags: z.array(z.string()).optional(),
    category: z.array(z.string()).optional(),
    toc: z.boolean().default(false),
  }),
  transform: async (document, context) => {
    const body = await compileMDX(context, document)
    return {
      ...document,
      body,
    }
  },
})

export default defineConfig({
  collections: [posts],
})

```

**Comment by sdorra** [comment]
It looks like there is a problem in the cli with the new version. I will check this. It looks like you are using Next.js, you could try the [Next integration](https://www.content-collections.dev/docs/integrations/next).

**Comment by sdorra** [comment]
@johanguse the issue should be fixed with the version 0.1.3 of the `@content-collections/mdx` package.
